packages/client3/pages/backer/all-artist-pools/index.page.tsx

```diff
@@ -21,36 +19,53 @@ function AllArtistPoolPage() {
   useSelectedSidebarItem("all-artist-pools");
   useLayoutTitle("All Artist Pools");
 
-  const { data, error } = useQuery(backerAllArtistPools);
-  const [openPoolData, setOpenPoolData] = useState<any[]>([]);
+  const { data } = useQuery(backerAllArtistPools);
   const router = useRouter();
-  /**
-   * Below code(useEffect) and it's usages need to be replaced by openTranchedPools when create pool insertion is done
-   */
-  useEffect(() => {
-    const fetchData = async () => {
-      const response = await axios.get(`/api/pool`);
-      const mappedData = Object.keys(response.data).map((key) => ({
-        id: key,
-        ...response.data[key],
-      }));
-      setOpenPoolData(mappedData);
-    };
-    fetchData();
-  }, []);
 
-  const openTranchedPools = data?.tranchedPools?.filter((tranchedPool: any) =>
-    (tranchedPool.juniorTranches[0].lockedUntil as BigNumber).isZero()
-  );
+  const openTranchedPools =
+    data?.tranchedPools?.filter((tranchedPool: any) =>
+      (tranchedPool.juniorTranches[0].lockedUntil as BigNumber).isZero()
+    ) || [];
 
-  const closedTranchedPools = data?.tranchedPools?.filter(
-    (tranchedPool: any) =>
-      !(tranchedPool.juniorTranches[0].lockedUntil as BigNumber).isZero()
-  );
+  const closedTranchedPools =
+    data?.tranchedPools?.filter(
+      (tranchedPool: any) =>
+        !(tranchedPool.juniorTranches[0].lockedUntil as BigNumber).isZero()
+    ) || [];
   const handleClick = (poolAddress: string) => {
     router.push(`/backer/pool/${poolAddress}`);
   };
 
+  const PoolComponent = (tranchedPool: any) => {
+    return (
+      <PoolCard
+        key={tranchedPool.id}
+        className="mb-10"
+        poolName={tranchedPool.name}
+        totalSuppliedAmount={{
+          token: SupportedCrypto.Usdc,
+          amount: tranchedPool?.juniorTranches[0].principalDeposited,
+        }}
+        totalGoalAmount={{
+          token: SupportedCrypto.Usdc,
+          amount: tranchedPool.creditLine.maxLimit,
+        }}
+        artistName={tranchedPool.borrower.name}
+        image={tranchedPool.borrower.logo}
```

Comment:
I believe we are not using the metadata under constants but rather using API to fetch meta data. To use it with gql Appolo rest has been integrated

Reply:
So we wouldn't be using @client declarator instead use @rest declarator to fetch data from API